app/tests/test_search.py

```diff
@@ -108,9 +117,10 @@ def verify_search_desktop_transferring_body_header_row(data):
 
     expected_row = (
         [
-            "Series reference",
-            "Consignment reference",
-            "File name",
+            "Found within",
```

Comment:
I was going to ask about this function and mention it seems odd to have found within and search results alongside these others but its no longer being used, so can be removed entirely!

However, it does highlight that we have no tests or assertions in existing tests covering the search Found within and search results part of the new results screen

Reply:
@anthonyhashemi will add two separate tests to differentiate between the two